### DISCUSSION TITLE
Telerik removal tool should check the version

### DISCUSSION THREAD

**Discussion by fablaser** [dnnsoftware/Dnn.Platform#5824] [General]
Hello, I purchased Telerik this year, anyway DNN suggest me to remove it. You should consider to check the telerik verrsion before showing the warning.

**Comment by mitchelsellers** [comment]
The exact version of Telerik installed by DNN Platform may not be the same for all installations (Depending on upgrade version, prior patch history etc.)

The removal tool at this time shows the version installed and the dependencies, there has not been a supported upgrade path to upgrade the DNN wrappers to use newer Telerik.

If you have patched to a new version and DNN is functional, you do not need to remove, but it is 100% untested in regards to all existing DNN Platform functionality that leverages Telerik

**Reply by Mostafa-Moafi** [reply]
If someone purchased the latest Telerik version so don't need to remove it. Is't right?
In the future, is it possible to upgrade Dnn to v10?

**Reply by mitchelsellers** [reply]
@Mostafa-Moafi yes, you will be able to upgrade, the forced removal will skip versions beyond the known issue versions.  There will be a warning that you are responsible for management
